packages/tools/pie-monorepo-utils/git-hooks/git-hooks-utils.js

```diff
@@ -5,13 +5,16 @@
  */
 function getTicketIdFromBranchName (branchName) {
     if (!branchName) throw new Error('A valid branch name wasn\'t provided');
-    const regexJiraTicket = /(^\w+?-\d+?)-\w.*/;
+    const regexJiraTicket = /(^\w+?-(\d{1,7}))-\w.*/;
```

Comment:
This max 7 digit JIRA ID was present in our commitlint config so added it here to for consistency.